### PR TITLE
fix assert statements

### PR DESCRIPTION
**Pull request by kain88-de**
Fixes #

@jbarnoud we had around 250 `assert_` calls left in the tests

Changes made in this Pull Request:
 -  replace `assert_` with plain assert

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: testsuite/MDAnalysisTests/utils/test_persistence.py [hunk 1]
```diff
@@ -125,9 +121,8 @@ def test_unpickle_reanchor(self, universe, pickle_str, ag):
         # now it goes back into the anchor list again
         universe.make_anchor()
         newag = cPickle.loads(pickle_str)
-        assert_array_equal(ag.indices, newag.indices)
```
Comment (richardjgowers) [inline testsuite/MDAnalysisTests/utils/test_persistence.py]: does this PR also do all `assert_array_equal` or just the ones you saw while doing assert_?

Comment (kain88-de) [reply testsuite/MDAnalysisTests/utils/test_persistence.py]: just that one. 

I think I can use the same technique to get rid of all `assert_array_equal` later today.

**Comment by orbeckst** [comment]
I just found two minor omissions and fixed them. Once travis passes, I will approve and squash/merge.

**Comment by orbeckst** [comment]
@jbarnoud I am currently resolving a rebase conflict in `test_groups.py` that looks like the following. My question: should all the stuff in the *replace `assert_` with plain assert* branch really be deleted (`make_groups_duplicated_and_scrambled` etc?)

```python
<<<<<<< HEAD
    def test_isdisjoint(self, groups):
        a, b, c, d, e = groups
        assert_(a.isdisjoint(e))
        assert_(e.isdisjoint(a))
        assert_(a.isdisjoint(d))
        assert_(d.isdisjoint(a))
        assert_(not a.isdisjoint(b))

    @pytest.mark.parametrize('left, right', itertools.chain(
=======
    @staticmethod
    def _test_isdisjoint(a, b, c, d, e):
        assert a.isdisjoint(e)
        assert e.isdisjoint(a)
        assert a.isdisjoint(d)
        assert d.isdisjoint(a)
        assert not a.isdisjoint(b)

    @staticmethod
    def make_groups(u, level):
        #   0123456789
        # a  ****
        # b    *****
        # c    **
        # e      ***
        # d empty
        #
        # None of the group start at 0, nor ends at the end. Each group
        # has a different size. The end of a slice is not the last element.
        # This increase the odds of catching errors.
        a = getattr(u, level)[1:5]
        b = getattr(u, level)[3:8]
        c = getattr(u, level)[3:5]
        d = getattr(u, level)[0:0]
        e = getattr(u, level)[5:8]
        return a, b, c, d, e

    @staticmethod
    def make_groups_duplicated_and_scrambled(u, level):
        # The content of the groups is the same as for make_groups, but the
        # elements can appear several times and their order is scrambled.
        a = getattr(u, level)[[1, 3, 2, 1, 2, 4, 4]]
        b = getattr(u, level)[[7, 4, 4, 6, 5, 3, 7, 6]]
        c = getattr(u, level)[[4, 4, 3, 4, 3, 3]]
        d = getattr(u, level)[0:0]
        e = getattr(u, level)[[6, 5, 7, 7, 6]]
        return a, b, c, d, e

    def test_groupbase_operators(self):
        n_segments = 10
        n_residues = n_segments * 5
        n_atoms = n_residues * 5
        u = make_Universe(size=(n_atoms, n_residues, n_segments))
        for level in ('atoms', 'residues', 'segments'):
            a, b, c, d, e = self.make_groups(u, level)
            yield self._test_len, a, b, c, d, e
            yield self._test_equal, a, b, c, d, e
            yield self._test_concatenate, a, b, c, d, e
            yield self._test_union, a, b, c, d, e
            yield self._test_intersection, a, b, c, d, e
            yield self._test_difference, a, b, c, d, e
            yield self._test_symmetric_difference, a, b, c, d, e
            yield self._test_issubset, a, b, c, d, e
            yield self._test_is_strict_subset, a, b, c, d, e
            yield self._test_issuperset, a, b, c, d, e
            yield self._test_is_strict_superset, a, b, c, d, e
            yield self._test_isdisjoint, a, b, c, d, e

    def test_groupbase_operators_duplicated_and_scrambled(self):
        n_segments = 10
        n_residues = n_segments * 5
        n_atoms = n_residues * 5
        u = make_Universe(size=(n_atoms, n_residues, n_segments))
        for level in ('atoms', 'residues', 'segments'):
            a, b, c, d, e = self.make_groups_duplicated_and_scrambled(u, level)
            yield self._test_len_duplicated_and_scrambled, a, b, c, d, e
            yield self._test_equal, a, b, c, d, e
            yield self._test_concatenate, a, b, c, d, e
            yield self._test_union, a, b, c, d, e
            yield self._test_intersection, a, b, c, d, e
            yield self._test_difference, a, b, c, d, e
            yield self._test_symmetric_difference, a, b, c, d, e
            yield self._test_issubset, a, b, c, d, e
            yield self._test_is_strict_subset, a, b, c, d, e
            yield self._test_issuperset, a, b, c, d, e
            yield self._test_is_strict_superset, a, b, c, d, e
            yield self._test_isdisjoint, a, b, c, d, e

    def test_only_same_level(self):
        def dummy(self, other):
            return True

        def failing_pairs(left, right):
            with pytest.raises(TypeError):
                _only_same_level(dummy)(left, right)

        def succeeding_pairs(left, right):
            assert _only_same_level(dummy)(left, right)

        _only_same_level = mda.core.groups._only_same_level
        u = make_Universe()

        components = (u.atoms[0], u.residues[0], u.segments[0])
        groups = (u.atoms, u.residues, u.segments)

>>>>>>> replace `assert_` with plain assert
```

**Comment by orbeckst** [comment]
I think I figured it out... I rebased and squashed, and should be good to go.

@jbarnoud if you want to take a quick glance?

**Comment by jbarnoud** [comment]
Looks OK.

**Pull request merged**